.github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: deploy
+
+on:
+  push:
+  pull_request:
```

Comment:
Why is it supposed to run on pull_request?

Reply:
Good point, we only need it on `schedule`.